Android/Jackson.Core/nuget/Jackson.Core.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Jackson.Core</id>
+    <title>Jackson Core for Xamarin Android</title>
+    <version>2.6.1</version>
+    <authors>Xamarin Inc.</authors>
+    <owners>Xamarin Inc.</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>This project contains core low-level incremental ("streaming") parser and generator abstractions used by Jackson Data Processor.</description>
+    <copyright>Copyright 2016 Xamarin Inc</copyright>
+    <projectUrl>https://github.com/xamarin/XamarinComponents/tree/master/Android/Jackson.Core</projectUrl>
+    <licenseUrl>https://github.com/xamarin/XamarinComponents/blob/master/Android/Jackson.Core/License.md</licenseUrl>
+  </metadata>
+  <files>  
+    <file src="output/Jackson.Core.dll" target="lib/MonoAndroid23" />
```

Comment:
I couldn't find any mention of lowest supported android version on the source repo, however I'm guessing since it's really not android specific code, this should be fine.  Has it been tested on 2.3 at all?